package.json

```diff
@@ -14,68 +14,69 @@
     },
     "dependencies": {
         "fast-equals": "^3.0.1",
-        "micro-memoize": "^4.0.9"
+        "micro-memoize": "^4.0.11"
```

Comment:
`fast-equals` has a breaking change for support of pre-ES2015 features, so it could not be updated.